streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/YarnQueueServiceImpl.java

```diff
@@ -0,0 +1,213 @@
+ * limitations under the License.
+ */
+
+package org.apache.streampark.console.core.service.impl;
+
+import org.apache.streampark.common.enums.ExecutionMode;
+import org.apache.streampark.console.base.domain.RestRequest;
+import org.apache.streampark.console.base.exception.ApiAlertException;
+import org.apache.streampark.console.core.bean.ResponseResult;
+import org.apache.streampark.console.core.entity.Application;
+import org.apache.streampark.console.core.entity.FlinkCluster;
+import org.apache.streampark.console.core.entity.YarnQueue;
+import org.apache.streampark.console.core.mapper.YarnQueueMapper;
+import org.apache.streampark.console.core.service.ApplicationService;
+import org.apache.streampark.console.core.service.FlinkClusterService;
+import org.apache.streampark.console.core.service.YarnQueueService;
+import org.apache.streampark.console.core.utils.YarnQueueLabelExpression;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import com.baomidou.mybatisplus.core.conditions.query.LambdaQueryWrapper;
+import com.baomidou.mybatisplus.core.metadata.IPage;
+import com.baomidou.mybatisplus.extension.plugins.pagination.Page;
+import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.Sets;
+import io.fabric8.kubernetes.client.utils.Utils;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.Nonnull;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.streampark.console.core.utils.YarnQueueLabelExpression.ERR_FORMAT_HINTS;
+
+@Slf4j
+@Service
+@Transactional(propagation = Propagation.SUPPORTS, readOnly = true, rollbackFor = Exception.class)
+public class YarnQueueServiceImpl extends ServiceImpl<YarnQueueMapper, YarnQueue>
+    implements YarnQueueService {
+
+  public static final String QUEUE_USED_FORMAT =
+      "Please remove the yarn queue for %s referenced it before %s.";
+  public static final String QUEUE_EXISTED_IN_TEAM_HINT =
+      "The queue label existed already. Try on a new queue label, please.";
+  public static final String QUEUE_EMPTY_HINT = "Yarn queue label mustn't be empty.";
+  public static final String QUEUE_AVAILABLE_HINT = "The queue label is available.";
+  public static final String NO_NEED_UPDATE_HINT = "Same data need not to update.";
+
+  @Autowired private ApplicationService applicationService;
+  @Autowired private FlinkClusterService flinkClusterService;
+
+  @Override
+  public IPage<YarnQueue> findYarnQueues(YarnQueue yarnQueue, RestRequest request) {
+    Utils.checkNotNull(yarnQueue, "Yarn queue query params mustn't be null.");
```

Comment:
We recommend use `org.apache.streampark.common.util.Utils.notNull(...)` 
Utils.checkNotNull import from `io.fabric8.kubernetes.client.utils.Utils`

Reply:
Nice catch👍~ & sorry for this error & updated it.